pages/user guides/tidal-vms.md

```diff
@@ -10,19 +10,19 @@ permalink: tidal-discovery-vm.html
 To help you get your Tidal Migrations discovery done easily and quickly you can use the Tidal Tools virtual machine image. This VM has the latest Tidal Migrations software and dependencies installed for you, so you can focus on getting things done.
 It can also perform discovery without any network connectivity to the public internet. To run a discovery in an 'offline' mode checkout [this guide](https://guides.tidalmg.com/tidal-offline.html).
 
-## Get Started
+## Ubuntu Server 18.04
 
-The image runs standard [Ubuntu Server (18.04.6)](https://cdimage.ubuntu.com/ubuntu/releases/18.04.6/release/) and has a CLI only interface. It comes with the latest version of [Tidal Tools](https://tidalmigrations.com/tidal-tools/), [Machine Stats](https://pypi.org/project/machine-stats/) and [DNS Tools](https://dnstools.ninja/) installed. 
+The image runs standard [Ubuntu Server (18.04.6)](https://cdimage.ubuntu.com/ubuntu/releases/18.04.6/release/) and has a CLI only interface. It comes with the latest version of [Tidal Tools](https://tidalmigrations.com/tidal-tools/), [Machine Stats](https://pypi.org/project/machine-stats/) and [DNS Tools](https://dnstools.ninja/) installed.
 
-It can be deployed into any virtual environment that is compaitble with OVA based images, and is comaptible with AMD64 or EM64T architectures. 
+It can be deployed into any virtual environment that is compatible with OVA based images, and is compatible with AMD64 or EM64T architectures.
```

Comment:
Nice catch! I use a vscode extension called `Code Spell Checker` personally to help with this, if you don't have it already. I can't spell to save my life 😅

Reply:
Yeah, I use the same. 😅

Reply:
I'd also recommend `markdownlint` extension with that.